src/components/NotFoundPanel.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <app-panel class="not-found-panel">
+    <p class="not-found-panel__paragraph">
+      <slot/>
+    </p>
+
+    <p class="not-found-panel__paragraph">
+      Maybe the
+      <app-link to="https://forum.aeternity.com/">
+        community
+      </app-link>
+      can help you.
+    </p>
+
+    <div class="not-found-panel__container">
+      <app-button
+        to="/"
+        class="not-found-panel__button">
+        Back to Dashboard
+      </app-button>
+    </div>
+  </app-panel>
+</template>
+
+<style scoped>
+.not-found-panel {
+  margin-top: var(--space-4);
+  padding: var(--space-4) var(--space-1);
+
+  @media (--desktop) {
+    padding: var(--space-4);
+  }
+
+  &__paragraph {
+    margin-bottom: var(--space-3);
+    @media (--desktop) {
+      &:last-of-type {
+        margin-bottom: var(--space-6);
+      }
+    }
+  }
+
+  &__container {
+    display: flex;
+    justify-content: center;
+
+    @media (--desktop) {
+      justify-content: flex-start;
+    }
+  }
+
+  &__button {
+    width: 215px;
+    padding: var(--space-3) var(--space-5);
+  }
+}
+</style>
+
+<style>
+.not-found-panel {
+  q {
```

Comment:
1. This is not written according to BEM methodology
2. I wouldn't say it's necessarily a quote by it's meaning and by style - quotes are usually italics. I would use the same as `search-error__query`. That's ideal scenarion
3. but in the case we want to save some code so we don't need to be that dogmatic. In that case, I suggest styling it in separate stylesheet styles/elements/_q.css